src/api/calendarEvents/index.js

```diff
@@ -80,7 +81,7 @@ export default class CalendarEvents extends Resource {
      * @apiExample {js} Example:
      *             gigwalk.calendarEvents.getForCustomer({...})
      */
-    getForCustomer(params: GetCalendarEventsForCustomerParams): APIPromise<Array<CalendarEvent>> {
+    getForCustomer(params: GetCalendarEventsForCustomerParams): APIPromise<Array<CalendarEventField>> {
```

Comment:
So the shape of `CalendarEvent` is completely wrong. Looks like it was a copy and paste from the Groups API. Can you keep this as `CalendarEvent` and fix the type declaration?